app/app/Http/Controllers/SignInController.php

```diff
@@ -39,6 +39,8 @@ public function authenticate(Request $request)
 			$email = $request->input('email');
 			if (BaseUser::authenticate($email, $request->input('password')))
 			{
+				if(!BaseUser::checkEmail($email))
+					return view('pages.signin',['email' => $email,'confirmmessage'=>'A verification code has ever been sent to your email '.$email.'. Check your email to confirm.']);
```

Comment:
Improve the language a little in your confirmmessage by removing the "ever" word and "your email".

Change:
"A verification code has ever been sent to your email "

To:
"A verification code has been sent to "